shipyard-controller/handler/projecthandler.go

```diff
@@ -303,6 +305,15 @@ func (ph *ProjectHandler) CreateProject(c *gin.Context) {
 			},
 			User: provisioningData.GitUser,
 		}
+	} else if automaticProvisioningURL == "" {
+		if params.GitCredentials == nil {
+			SetBadRequestErrorResponse(c, fmt.Sprintf(common.InvalidRequestFormatMsg, "cannot create project without git credentials"))
+			return
+		}
```

Comment:
I would not add this extra assumption. If the credentials are necessary, then the resource-svc will fail